src/Dobble/DobbleCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
php formatting is weird. Run [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) on your code to see the usual coding standards (refs PSR/1). It'd be a good idea to add php-cs-fixer to your Travis build, too (e.g. https://github.com/fzaninotto/Faker/blob/master/Makefile#L4-L6)